superface/super.json

```diff
@@ -191,8 +191,14 @@
         },
         "azure-maps": {
           "file": "../grid/address/geocoding/maps/azure-maps.suma"
+        },
+        "tomtom": {
+          "file": "../grid/address/geocoding/maps/tomtom.suma"
         }
-      }
+      },
+      "priority": [
+        "tomtom"
+      ]
```

Comment:
Preferably remove the priority array please.

Reply:
fixed, thanks!